### PR TITLE
Revised INT8 Inference doc

### PR DESCRIPTION
**Pull request by AlexKoff88**
Please take a look at these changes. I removed two sentences about LPT because they just don't make sense. I suggest writing a separate detailed doc about this module and refer to it from this one.

### CODE REVIEW

File: docs/IE_DG/Int8Inference.md [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
-2. [Low-Precision 8-bit Integer Inference Workflow](#low-precision-8-bit-integer-inference-workflow)
-3. [Prerequisites](#prerequisites)
-4. [Inference](#inference)
-5. [Results analysis](#results-analysis)
+## Introduction
+The goal of this document is to describe some aspects of low-precision inference using OpenVINO Inference Engine (IE). 
+Currently, many Intel HW devices support 8-bit arithmetic out-of-the-box. It allows boosting inference performance 
+significantly by loading more data into a single processor instruction and executing more arithmetical operations at
+one time. The real performance improvement from 8-bit inference depends on the HW but theoretically it can achieve **4X**
+ compared with FP32 execution and **2X** with FP16 at some cost of accuracy which is negligible in many cases.
```
Comment (MaximProshin) [inline docs/IE_DG/Int8Inference.md:8]: Is it really needed: "at some cost of accuracy which is negligible in many cases"? This is not about the quantization but rather FP vs INT8 here and I think it's useless to talk about the accuracy here.

File: docs/IE_DG/Int8Inference.md [hunk 2]
```diff
@@ -1,15 +1,35 @@
...
-## Supported devices
+## Low-Precision 8-bit Integer Inference Workflow
+Low-precision workflow consists of two basic steps:
+- *Model preparation*. This step includes model quantization using the tools from OpenVINO ecosystem such as:
+  - [Post-training Optimization Toolkit](@ref pot_README)
+  - [Neural Network Compression Framework](https://github.com/openvinotoolkit/nncf)
+  Please refer to the [Low Precision Optimization Guide](@ref pot_docs_LowPrecisionOptimizationGuide) 
+for more details about the model optimization workflow. 
+- *Low-precision 8-bit inference*. At this step, the quantized model undergoes a set of transformations (the so-called 
+`Low Precision Transformations`) that convert it
```
Comment (MaximProshin) [inline docs/IE_DG/Int8Inference.md:18]: convert=>converts

File: docs/IE_DG/Int8Inference.md [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
+- *Model preparation*. This step includes model quantization using the tools from OpenVINO ecosystem such as:
+  - [Post-training Optimization Toolkit](@ref pot_README)
+  - [Neural Network Compression Framework](https://github.com/openvinotoolkit/nncf)
+  Please refer to the [Low Precision Optimization Guide](@ref pot_docs_LowPrecisionOptimizationGuide) 
+for more details about the model optimization workflow. 
+- *Low-precision 8-bit inference*. At this step, the quantized model undergoes a set of transformations (the so-called 
+`Low Precision Transformations`) that convert it 
+to the representations that are used by the OpenVINO™ plugins to map the quantized operations to low-level low-precision 
+inference primitives. When you pass the quantized IR to the OpenVINO™ plugin, the plugin automatically recognizes it as
+a quantized model. An indicator of it is [FakeQuantize](../ops/quantization/FakeQuantize_1.md) operation that provides
```
Comment (MaximProshin) [inline docs/IE_DG/Int8Inference.md:21]: An indicator => The indicator

File: docs/IE_DG/Int8Inference.md [hunk 4]
```diff
@@ -1,15 +1,35 @@
...
+  Please refer to the [Low Precision Optimization Guide](@ref pot_docs_LowPrecisionOptimizationGuide) 
+for more details about the model optimization workflow. 
+- *Low-precision 8-bit inference*. At this step, the quantized model undergoes a set of transformations (the so-called 
+`Low Precision Transformations`) that convert it 
+to the representations that are used by the OpenVINO™ plugins to map the quantized operations to low-level low-precision 
+inference primitives. When you pass the quantized IR to the OpenVINO™ plugin, the plugin automatically recognizes it as
+a quantized model. An indicator of it is [FakeQuantize](../ops/quantization/FakeQuantize_1.md) operation that provides 
+the quantization parameters and scheme. If you pass a quantized model to the plugin that does not support 8-bit
+inference but supports all operations from the model, the model is inferred in the floating-point precision that this
+plugin supports.
```
Comment (MaximProshin) [inline docs/IE_DG/Int8Inference.md:24]: I think this is confusing/unclear: "the model is inferred in the floating-point precision that this
plugin supports". At least I don't understand what you mean here. If it supports FP32 and FP16, how the model will be inferred? Maybe it's better to just say "the model is inferred in the floating-point precision".

**Comment by ilya-lavrenov** [comment]
@AlexKoff88 is it still valid or can be closed?

**Pull request closed without merging**